src/javascript/app/Stores/Modules/Trading/trade-store.js

```diff
@@ -187,7 +195,7 @@ export default class TradeStore extends BaseStore {
             this.currency           = this.root_store.client.currency;
             this.initial_barriers   = { barrier_1: this.barrier_1, barrier_2: this.barrier_2 };
 
-            if (!this.symbol) {
+            if (!this.symbol || this.selectedSymbolExchangeStatus(active_symbols) !== 1) {
```

Comment:
If there is no symbol, then wouldn't `selectedSymbolExchangeStatus` always return `undefined`?